recipes/cif2cell/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "cif2cell" %}
+{% set version = "1.2.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://sourceforge.net/projects/cif2cell/files/{{ name }}-{{ version }}.tar.gz
+  sha256: 2268a2d6d26650433a1c5e51c98e400cf92e458036879e231088555d75e3e8bf
+
+build:
+  noarch: python
```

Comment:
I don't think scripts currently support noarch. You might try it without noarch.

Reply:
Looks like remove "noarch" fixed the previous Windows error, but there is another one now, "OSError: [WinError 123] The filename, directory name, or volume label syntax is incorrect: 'C:\\bld\\cif2cell_1545421841878\\work_moved_cif2cell-1.2.10-py<3_0_win-64_main_build_loop'".